tests/unit/options.js

```diff
@@ -194,11 +194,15 @@
 
       assert.equal(menu().parent().find(".ui-multiselect-menu").outerWidth(), width, 'width after opening, property set to ' + width);
 
-      // change height and re-test
+      // change width and re-test
       width = 300;
       el.multiselect("option", "menuWidth", width).multiselect('refresh');
       assert.equal(menu().parent().find(".ui-multiselect-menu").outerWidth(), width, 'changing value through api to ' + width);
 
+      width = "3in";
+      el.multiselect("option", "menuWidth", width).multiselect('refresh');
+      assert.equal(menu().parent().find(".ui-multiselect-menu").outerWidth(), 3 * 96.0, 'menuWidth supports strings suffixed with various units as well as integer px values');
```

Comment:
Can you update the assert message here with the `in` unit?

Reply:
ok